SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -38,16 +38,34 @@ bool GenerateWorkerAssemblies()
 	return ExitCode == SpatialGDKServicesConstants::ExitCodeSuccess;
 }
 
-bool GenerateWorkerJson()
+FString GetWorkerJsonPath()
 {
 	const FString WorkerJsonDir = FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("workers/unreal"));
 
 	FString Filename = FString(TEXT("spatialos.")) + *AutomationWorkerType.ToString() + FString(TEXT(".worker.json"));
 	FString JsonPath = FPaths::Combine(WorkerJsonDir, Filename);
+	return JsonPath;
+}
+
+bool GenerateWorkerJson()
+{
+	FString JsonPath = GetWorkerJsonPath();
 	if (!FPaths::FileExists(JsonPath))
 	{
-		bool bRedeployRequired = false;
-		return GenerateDefaultWorkerJson(JsonPath, bRedeployRequired);
+		bool bOutRedeployRequired = false;
+		return GenerateDefaultWorkerJson(JsonPath, bOutRedeployRequired);
+	}
+
+	return true;
+}
+
+bool DeleteWorkerJson()
```

Comment:
Is this all worker jsons? Think we should be explicit

Reply:
Also why do we call them jsons and not worker configs?

Reply:
too many configs

Reply:
I think this one may be fine, it's local to the test, and in an anonymous namespace

Reply:
It's basically to pair with `GenerateWorkerJson` just above (also in anonymous namespace)